### PR TITLE
Fixing some typos.

### PR DESCRIPTION
**Pull request by armfazh**

### CODE REVIEW

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -308,19 +308,19 @@ prime-order group.
 - Identity(): Outputs the identity element of the group (i.e. `I`).
 - Serialize(A): A member function of `GG` that maps a group element `A`
   to a unique byte array `buf`.
-- Deserialize(buf): A member function of `GG` that maps a byte array
-  `buf` to a group element `A`.
+- Deserialize(buf): A member function of `GG` that maps a valid byte array
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:311]: What does "valid" mean?

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -444,17 +444,16 @@ contexts are below:
 ~~~
 def SetupVerifiableServer(suite, skS, pkS):
   contextString = I2OSP(modeVerifiable, 1) || I2OSP(suite.ID, 2)
-  return VerifiableServerContext(contextString, skS), pkS
+  return VerifiableServerContext(contextString, skS, pkS)
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:447]: This wasn't a typo -- it's mean to output both the context and the public key, so the latter can be distributed to clients.

```suggestion
  return VerifiableServerContext(contextString, skS), pkS
```

File: draft-irtf-cfrg-voprf.md [hunk 3]
```diff
@@ -571,7 +570,7 @@ Output:
 
   opaque output<1..2^16-1>
 
-def VerifyFinalize(skS, input, info):
+def FullEvaluate(skS, input, info):
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:573]: Hah, oops.

File: draft-irtf-cfrg-voprf.md [hunk 4]
```diff
@@ -988,7 +987,11 @@ and ristretto255. See {{cryptanalysis}} for related discussion.
 - Group: P-521 (secp521r1) {{x9.62}}
   - HashToGroup(): P521_XMD:SHA-512_SSWU_RO\_
     {{!I-D.irtf-cfrg-hash-to-curve}} with DST =
+<<<<<<< HEAD
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md]: Merge conflict -- please fix.

File: draft-irtf-cfrg-voprf.md [hunk 5]
```diff
@@ -670,25 +669,25 @@ def GenerateProof(skS, pkS, blindToken, element)
   blindTokenList = [blindToken]
   elementList = [element]
 
-  (a1, a2) = ComputeComposites(pkS, blindTokenList, elementList)
+  a = ComputeComposites(pkS, blindTokenList, elementList)
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:672]: Why did we make this change?

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-irtf-cfrg-voprf.md (hunk 1 above)
Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:311]: valid means the output of Serialized,

File: draft-irtf-cfrg-voprf.md (hunk 2 above)
Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:447]: ok, reverting

File: draft-irtf-cfrg-voprf.md (hunk 5 above)
Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:672]: ComputeComposites returns an array/vector

File: draft-irtf-cfrg-voprf.md (hunk 1 above)
Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md]: Can you clarify that?

File: draft-irtf-cfrg-voprf.md (hunk 5 above)
Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:673]: Okay, and is this reflected in the Sage code?

File: draft-irtf-cfrg-voprf.md (hunk 1 above)
Comment (armfazh) [reply draft-irtf-cfrg-voprf.md]: yes, see the updated version

File: draft-irtf-cfrg-voprf.md [hunk 6]
```diff
@@ -309,18 +309,19 @@ prime-order group.
...
   computationally difficult to reverse the mapping. Examples of hash to
   group functions satisfying this property are described for prime-order
   (sub)groups of elliptic curves, see {{!I-D.irtf-cfrg-hash-to-curve}}.
 - HashToScalar(x): A member function of `GG` that deterministically maps
-  an array of bytes `x` to a random element in GF(p). A recommended method
+  an array of bytes `x` to an element in GF(p). A recommended method
   for its implementation is instantiating the hash to field function,
   defined in {{!I-D.irtf-cfrg-hash-to-curve}} setting the target field to GF(p).
-- RandomScalar(): A member function of `GG` that generates a random,
+- RandomScalar(): A member function of `GG` that choses at random a
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:324]:
```suggestion
- RandomScalar(): A member function of `GG` that chooses at random a
```

**Review by chris-wood — APPROVED** [review]

**Pull request merged**